src/fu-util.c

```diff
@@ -1705,6 +1705,19 @@ fu_util_report_history(FuUtilPrivate *priv, gchar **values, GError **error)
 				continue;
 			}
 		}
+
+		/* needs an extra action */
+		if (fwupd_device_has_flag(dev, FWUPD_DEVICE_FLAG_NEEDS_ACTIVATION)) {
+			fu_console_print(
+			    priv->console,
+			    /* TRANSLATORS: %1 is a device name, e.g. "ThinkPad Universal
+			     * ThunderBolt 4 Dock" and %2 is "fwupdmgr activate" */
+			    _("%s is pending activation; use %s to complete the update."),
+			    fwupd_device_get_name(dev),
+			    "fwupdmgr activate");
```

Comment:
Is it worth using a split up $0 here?

Reply:
not sure i follow; you mean make it into two sentences?

Reply:
I mean for the command used to launch in case of a renamed binary or not part of PATH.

It's probably overkill.

Reply:
Nah, it's a good idea! We do it for `g_autofree gchar *cmd = g_strdup_printf("%s sync", g_get_prgname());` already. Give me 1 min.